### PR TITLE
Whatsnew v0.16

### PR DESCRIPTION
**Pull request by pp-mo**
Writeups for changes since 0.15.0, since we don't seem to have updated the whatsnew with 0.15.1 changes.
This is intended to be almost-ready for release, but...
  * I am assuming that #223 is going in
  * the actual release date will need writing in

I have tried to use style closer to the newer Iris whatsnews.  
Please highlight if more could be done in that way.

**temporary docs-build can be viewed [here](https://pp-iris-grib.readthedocs.io/en/whatsnew_16/ref/release_notes.html)**

### CODE REVIEW

File: docs/ref/release_notes.rst [hunk 1]
```diff
@@ -1,6 +1,117 @@
...
+* `@m1dr <https://github.com/m1dr>`_ added support for loading statistical
+  fields, as encoded in production definition template 3.8, even when the
+  "interval time increment" value is not specified (i.e. set to "missing").
+  See `PR#206 <https://github.com/SciTools/iris-grib/pull/206>`_ .
+
+* `@pp-mo <https://github.com/pp-mo>`_ ported some tests from Iris, which test
+  grib saving of data loaded from other formats.
+  See `PR#213 <https://github.com/SciTools/iris-grib/pull/213>`_ .
+  All grib-dependent testing is now contained in iris-grib : **There are no
+  remaining tests in Iris which use grib.**
```
Comment (trexfeathers) [inline docs/ref/release_notes.rst:32]: PR reference should be at the end

**Review by trexfeathers — CHANGES_REQUESTED** [review]
Looks pretty good so far @pp-mo 👌

I've noted a couple of points for you to address. Apart from those, I'm satisfied that you've covered everything that's happened since `0.15.0`.

This is low importance but if you wanted to strictly match the [Iris "What's New" structure](https://scitools-iris.readthedocs.io/en/latest/developers_guide/documenting/whats_new_contributions.html#writing-a-contribution) then the PR references should be in brackets with no connective words and no need for a `.`.

File: docs/ref/release_notes.rst (hunk 1 above)
Comment (pp-mo) [reply docs/ref/release_notes.rst:32]: It isn't really about this PR, just this one happened to be the last outstanding.  
I might just add it as a separate bullet.

File: docs/ref/release_notes.rst [hunk 2]
```diff
@@ -1,6 +1,117 @@
...
+
+Internal
+^^^^^^^^
+
+* `@lbdreyer <https://github.com/lbdreyer>`_ and
+  `@pp-mo <https://github.com/pp-mo>`_ ported various grib-specific tests from
+  Iris.
+  See `PR#191 <https://github.com/SciTools/iris-grib/pull/191>`_ ,
+  `PR#192 <https://github.com/SciTools/iris-grib/pull/192>`_ ,
+  `PR#193 <https://github.com/SciTools/iris-grib/pull/193>`_ ,
```
[the remark this answers is not in the document]
Comment (pp-mo) [reply docs/ref/release_notes.rst:99]: Good spot!

**Comment by pp-mo** [comment]
Thanks @trexfeathers.
Attended to those things, I think, plus some new spots of my own.
Also re-spun the sample docs build (see link in head).
**Please re-review.** :grin:

**Review by trexfeathers — APPROVED** [review]

**Comment by trexfeathers** [comment]
Thanks @pp-mo !

**Pull request merged**